src/bp/sdk/botpress.d.ts

```diff
@@ -523,7 +523,7 @@ declare module 'botpress/sdk' {
 
     export type ProgressReporter = (botId: string, message: string, trainSession: TrainingSession) => void
 
-    export type EntityType = 'system' | 'pattern' | 'list'
+    export type EntityType = 'system' | 'pattern' | 'list' | 'complex' // TODO: Add the notion of Utterance Placeholder instead of adding "Complex" as an entity type here (synonyms and variables)
```

Comment:
So complex is only a temporary concept? At least we don't want to expose it right?